runtime/tests/interpreter/interpreter_test.go

```diff
@@ -6594,14 +6592,15 @@ func TestInterpretResourceMoveInArrayAndDestroy(t *testing.T) {
           destroy foos
           return bar
       }
-    `)
-
-	AssertValuesEqual(
-		t,
-		inter,
-		interpreter.NewUnmeteredIntValueFromInt64(0),
-		inter.Globals.Get("destroys").GetValue(),
-	)
+    `, ParseCheckAndInterpretOptions{
+		Config: &interpreter.Config{
+			OnEventEmitted: func(inter *interpreter.Interpreter, locationRange interpreter.LocationRange, event *interpreter.CompositeValue, eventType *sema.CompositeType) error {
```

Comment:
Here and in the rest of the PR: replace unuesed parameters with `_`

```suggestion
			OnEventEmitted: func(_ *interpreter.Interpreter, _ interpreter.LocationRange, _ *interpreter.CompositeValue, _ *sema.CompositeType) error {
```